tests/test_project.py

```diff
@@ -990,8 +990,8 @@ def test_export_custom_path_string_modify_tree_flat(self):
         for i in range(10):
             for j in range(2):
                 for k in range(2):
-                    for l in range(2):
-                        self.project.open_job(dict(a=i, b=j, c=k, d=l)).init()
+                    for m in range(2):
```

Comment:
I know these are just tests, but IMHO going from `l` to `m` here is confusing since we are going `i, j, k, l`. My preference would be to add some sort of #noqa to make flake8 happy.

Reply:
Or descriptive names like `a_value`, `b_value`, `c_value`, `d_value` that correspond to the state point.

Reply:
There are several places where this is repeated. Hence, instead of using #noqa everywhere, I think I should go with changing all the names.